core/client-internal/src/test/java/tachyon/client/block/TachyonBlockStoreTest.java

```diff
@@ -97,7 +96,6 @@
   public void before() throws Exception {
     mTestFile = mTestFolder.newFile("testFile");
 
-    ClientContext.reset();
```

Comment:
in this `TachyonBlockStoreTest`, `BlockStoreContext` was modified. don't we need to reset the `ClientContext` in `after`?